spec/features/following_users_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+feature "User follows a user and unfollow" do
+  scenario "follow a user" do
+    comedies = Fabricate(:category, name: "Comedies")
+    video = Fabricate(:video, category: comedies)
+    user1 = Fabricate(:user)
+    user2 = Fabricate(:user)
+    review = Fabricate(:review, video: video, user: user2)
+    sign_in(user1)
+
+    click_video(video)
+    click_link(user2.full_name)
+    expect(page).to have_content "#{user2.full_name}'s video collections"
+
+    click_link "Follow"
+    expect(page).to have_content "People I Follow"
```

Comment:
Consider moving expectations like this to custom made expectation methods that you create yourself. You can define these at the bottom of a feature spec.